src/Microsoft.VisualBasic/tests/ConversionsTests.cs

```diff
@@ -5,83 +5,2952 @@
+            yield return new object[] { (UShortEnum)ushort.MinValue, (sbyte)0 };
+            yield return new object[] { (UShortEnum)1, (sbyte)1 };
+
+            // short.
+            yield return new object[] { (short)(-1), (sbyte)(-1) };
+            yield return new object[] { (short)0, (sbyte)0 };
+            yield return new object[] { (short)1, (sbyte)1 };
+            yield return new object[] { (ShortEnum)(-1), (sbyte)(-1) };
+            yield return new object[] { (ShortEnum)0, (sbyte)0 };
+            yield return new object[] { (ShortEnum)1, (sbyte)1 };
+
+            // uint.
+            yield return new object[] { uint.MinValue, (sbyte)0 };
+            yield return new object[] { (uint)1, (sbyte)1 };
+            yield return new object[] { (UIntEnum)uint.MinValue, (sbyte)0 };
+            yield return new object[] { (UIntEnum)1, (sbyte)1 };
+
+            // int.
+            yield return new object[] { -1, (sbyte)(-1) };
+            yield return new object[] { 0, (sbyte)0 };
+            yield return new object[] { 1, (sbyte)1 };
+            yield return new object[] { (IntEnum)(-1), (sbyte)(-1) };
+            yield return new object[] { (IntEnum)0, (sbyte)0 };
+            yield return new object[] { (IntEnum)1, (sbyte)1 };
+
+            // ulong.
+            yield return new object[] { ulong.MinValue, (sbyte)0 };
+            yield return new object[] { (ulong)1, (sbyte)1 };
+            yield return new object[] { (ULongEnum)ulong.MinValue, (sbyte)0 };
+            yield return new object[] { (ULongEnum)1, (sbyte)1 };
+
+            // long.
+            yield return new object[] { (long)(-1), (sbyte)(-1) };
+            yield return new object[] { (long)0, (sbyte)0 };
+            yield return new object[] { (long)1, (sbyte)1 };
+            yield return new object[] { (LongEnum)(-1), (sbyte)(-1) };
+            yield return new object[] { (LongEnum)0, (sbyte)0 };
+            yield return new object[] { (LongEnum)1, (sbyte)1 };
+
+            // float.
+            yield return new object[] { (float)(-1), (sbyte)(-1) };
+            yield return new object[] { (float)0, (sbyte)0 };
+            yield return new object[] { (float)1, (sbyte)1 };
+
+            // double.
+            yield return new object[] { (double)(-1), (sbyte)(-1) };
+            yield return new object[] { (double)0, (sbyte)0 };
+            yield return new object[] { (double)1, (sbyte)1 };
+
+            // decimal.
+            yield return new object[] { (decimal)(-1), (sbyte)(-1) };
+            yield return new object[] { (decimal)0, (sbyte)0 };
+            yield return new object[] { (decimal)1, (sbyte)1 };
+
+            // bool.
+            yield return new object[] { true, (sbyte)(-1) };
+            yield return new object[] { false, (sbyte)0 };
+
+            // null.
+            yield return new object[] { null, (sbyte)0 };
+
+            // string.
+            yield return new object[] { "-1", (sbyte)(-1) };
```

Comment:
same comment as above. remove the chunk and use the already defined function as member data

Reply:
Fixed